unicodetools/data/ucd/dev/PropList.txt

```diff
@@ -1044,6 +1045,8 @@ FFE3          ; Diacritic # Sk       FULLWIDTH MACRON
 10AE5..10AE6  ; Diacritic # Mn   [2] MANICHAEAN ABBREVIATION MARK ABOVE..MANICHAEAN ABBREVIATION MARK BELOW
 10D22..10D23  ; Diacritic # Lo   [2] HANIFI ROHINGYA MARK SAKIN..HANIFI ROHINGYA MARK NA KHONNA
 10D24..10D27  ; Diacritic # Mn   [4] HANIFI ROHINGYA SIGN HARBAHAY..HANIFI ROHINGYA SIGN TASSI
+10D4E         ; Diacritic # Lo       GARAY VOWEL LENGTH MARK
```

Comment:
The Garay length mark does not look like a diacritic to me.
It looks more like a U+30FC KATAKANA-HIRAGANA PROLONGED SOUND MARK ... which is both Extender and, to my surprise, also Diacritic.
And, that one is gc=Lm while the Garay length mark is gc=Lo. There are many Lm Diacritics but few Lo ones.

So:
- Should the Garay length mark be gc=Lm?
- Why is U+30FC a Diacritic?

@Ken-Whistler advice?

Reply:
For context:

[Diacritic](https://www.unicode.org/reports/tr44/#Diacritic) |	B	| I	| Characters that linguistically modify the meaning of another character to which they apply. Some diacritics are not combining characters, and some combining characters are not diacritics.
-- | -- | -- | --

Reply:
I know, I read that, and the glossary entry. But without contrasting examples, and without knowing how this property is used, I still find these definitions fuzzy. If Diacritic covers any sort of "to which they apply" then there is little difference with gc=Lm. These two length marks certainly don't _look like_ what I think of as a diacritic.

Reply:
> These two length marks certainly don't _look like_ what I think of as a diacritic.

I don’t think the appearance actually matters to diacriticness; the modification here is linguistic, not graphical stickingoutness.